include/model.php

```diff
@@ -65,6 +65,30 @@ public function __construct( &$options ) {
 		add_filter( 'language_description', '__return_empty_string' );
 	}
 
+	/**
+	 * Checks if there are languages or not.
+	 *
+	 * @since 3.3
+	 *
+	 * @return bool True if there are, false otherwise.
+	 */
+	public function has_languages() {
+		if ( false !== $this->cache->get( 'languages' ) ) {
+			return true;
+		}
+
+		if ( false !== get_transient( 'pll_languages_list' ) ) {
+			return true;
+		}
+
+		$post_languages = get_terms( array( 'taxonomy' => 'language', 'hide_empty' => false ) );
```

Comment:
I suggest to use the same arguments that the ones used in `get_languages_list()`:

```suggestion
		$post_languages = get_terms( array( 'taxonomy' => 'language', 'hide_empty' => false, 'orderby' => 'term_group' ) );
```

The reason:
`get_terms()` uses `WP_Term_Query->query( $args )`, which uses [a cache system based on the arguments](https://github.com/WordPress/WordPress/blob/5097c83eac3f1446bfa8c31c4ae3d7a858df76ba/wp-includes/class-wp-term-query.php#L761-L764). So if `get_languages_list()` is used after `has_languages()`, WP won't do another query to the DB. And this doesn't add any complexity to the query itself.